src/complete.js

```diff
@@ -0,0 +1,39 @@
+/* eslint import/no-cycle: [0, { maxDepth: 2 }] */
+import { addTaskToStore } from './add-remove.js';
+// check the task and add to store
+const completeStateChange = (taskArray) => {
+  const checkedInputElements = document.getElementsByClassName('check-input');
+  for (let i = 0; i < checkedInputElements.length; i += 1) {
+    checkedInputElements[i].addEventListener('change', (event) => {
+      taskArray[event.target.getAttribute('check-id') - 1].completed = event.target.checked;
+      addTaskToStore(taskArray);
+    });
+  }
+};
+
+const addCheckedAttFromStore = (tasks) => {
+  for (let i = 0; i < tasks.length; i += 1) {
+    const checkedInputElements = document.getElementsByClassName('check-input');
+    if (tasks[i].completed) {
+      checkedInputElements[i].setAttribute('checked', '');
+      //   console.log(checkedInputElements[i])
+    } else {
+      checkedInputElements[i].removeAttribute('checked');
+      //    console.log(checkedInputElements[i])
+    }
+  }
+};
```

Comment:
- [ ] Please fix the following behavior that occurs whenever the user adds a task to the list.

https://user-images.githubusercontent.com/62290545/172663828-8ae0f29c-682a-4496-8dc3-403457000119.mp4